src/service/company_service.rs

```diff
@@ -259,12 +298,18 @@ impl CompanyServiceApi for CompanyService {
         Ok(())
     }
 
-    async fn add_signatory(&self, id: &str, signatory_node_id: String) -> Result<()> {
+    async fn add_signatory(
+        &self,
+        id: &str,
+        signatory_node_id: String,
+        timestamp: i64,
```

Comment:
we should use u64 everywhere for timestamp. Lets do some backlog task for it?

Reply:
Yeah, on that note, we maybe could create a task to rethink the way we do these timestamps, since the current solution with the external atomic time service isn't optimal. :thinking:

Reply:
lets do it

Reply:
Added https://github.com/BitcreditProtocol/E-Bill/issues/274